app/src/main/java/net/gsantner/markor/ui/DraggableScrollbarScrollView.java

```diff
@@ -77,15 +78,14 @@ protected void onAttachedToWindow() {
         if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.JELLY_BEAN_MR1) {
             _ltr = getLayoutDirection() == View.LAYOUT_DIRECTION_LTR;
         }
+        _scrollbarWidth = getVerticalScrollbarWidth();
     }
 
-    public void enableFastScroll() {
-        _fastScrollEnabled = true;
+    public void set_fastScrollEnabled(boolean _fastScrollEnabled) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The instance method name 'set_fastScrollEnabled' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/gsantner/markor/pullRequest?prid=5576245)